_includes/documentation-card.html

```diff
@@ -0,0 +1,14 @@
+<div class="col-12 col-md-3 mb-3 mb-md-3">
+<a class="text-decoration-none" href="{{ include.item.url | replace:'.html','' }}">
+    <div class="card shadow-sm project-card">
+    <i class="{{ include.item.icon }} size-36 text-main mb-3"></i>
+    <h4 class="text-main weight-700">
+    {% if documentation.level == "1" %}
+        {% t documentation.categories.{{ include.item.group }}.title %}
+    {% elsif documentation.level == "2" %}
+        {% t documentation.categories.{{ include.item.group }}.{{ include.item.slug }} %}
+    {% endif %}
```

Comment:
Could be a lot less complex

```suggestion
    {% t {{ include.item.title }} %}
```

Reply:
Ah, yes, apparently forgot to copy that from my test repo to this branch.